### DISCUSSION TITLE
how to re-install ravendb completely

### DISCUSSION THREAD

**Discussion by Vincent026** [ravendb/ravendb#16831] [Q&A] [answered]
I am new to ravendb. I chose a secure install (on WIndows 10) and received a zip file with the client cert (I think). Because I can't make it work I want to reinstall ravendb completely, and choose the unsecure route in the wizard instead. If I run uninstall-service.ps1 and then setup-as-service.ps1 it uses cached/known settings and I end up with the same situation where I can't open studio, because of the unknown client cert. How do I remove/re-install ravendb as if it was a brand new machine?
![raven](https://github.com/ravendb/ravendb/assets/7458571/23b0b667-0b3f-481c-bc87-eda10907fc1b)

**Comment by ayende** [comment]
The credentials are cached by the browser. Try accessing it from an incognito winodw

**Reply by Vincent026** [reply]
Thanks for your answer but I get the same error in an incognito window. Therefore I want to start from scratch, but the installer skips the wizard altogether now.

**Reply by Vincent026** [reply, marked as answer]
I managed to get the wizard back by deleting all files and unzip the installer. I ran the installer and have an unsecure setup, what I wanted. That is fine for now.
